04-vector-raster-integration-advanced.Rmd

```diff
@@ -277,6 +293,13 @@ plot(plot.locationSp_HARV,
 
 ```
 
```

Comment:
Try this:

Refer to the first figure in this lesson which shows the extents of
all the spatial objects including our the plot locations layer. Notice that the plot locations extent (black) extends further west than the extent of our canopy height model raster (green). When we plot the data, R will set the **plot** extent to the extent of the first layer plotted - in this case, the raster canopy height model. Thus any points that fall outside of that extent (for example our additional vegetation plot) will not be displayed in the plot.

****
Megan - the problem with the text is that we didn't crop the CHM to the plots - we cropped it to the AOI boundary and that extent is different too. Have a close look at what we are doing in this lesson. Thus, the cropping part is not relevant. what is relevant is that there are different extents and when you have different extents, R will take the extent of the first object UNLESS you specify the plot extent explicetly using xlim, xlim. Does that make sense?